Configure

```diff
@@ -528,6 +528,7 @@ my @disablables = (
     "thread-pool",
     "threads",
     "tls",
+    "tls1_3-integrity-only_ciphers",
```

Comment:
Instead of this very narrow build option I'd prefer an option `integrity-only-ciphers` that would remove all ciphersuites with eNULL.

We will also need a new test case in runchecker-merge-ci workflow that disables the option (and please test that it passes).